engine/queue/score-queue-impl/src/main/java/org/openscore/engine/queue/services/ExecutionQueueServiceImpl.java

```diff
@@ -73,6 +73,15 @@ public void enqueue(List<ExecutionMessage> messages) {
 				stateMessages.add(msg);
 			}
 		}
+
+        if (CollectionUtils.isNotEmpty(listeners)) {
+            long t = System.currentTimeMillis();
```

Comment:
you know there is a normal StopWatch for that:
[https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/time/StopWatch.html]